# Engine id to dispose
string id
---
# Success == 1, Failure == 0
int32 success
